OpenRA.Game/Platform.cs

```diff
@@ -94,12 +94,26 @@ static string GetSupportDir()
 			return dir + Path.DirectorySeparatorChar;
 		}
 
-		public static string GameDir { get { return AppDomain.CurrentDomain.BaseDirectory; } }
+		public static string GameDir
+		{
+			get
+			{
+				var gameDir = AppDomain.CurrentDomain.BaseDirectory;
+
+				// BUG: In Game.cs we get GameDir with trailing DirectorySeparator, but not in PlatformTest.cs
+				if (!gameDir.EndsWith(Path.DirectorySeparatorChar.ToString()))
```

Comment:
My first suggestion has the potential to behave badly with unicode paths.  Best to stick with `EndsWith` but add the `StringComparison` argument as I noted in my second suggestion.